security/pkg/nodeagent/test/cert_rotation/rotation_test.go

```diff
@@ -92,8 +92,8 @@ func TestCertRotation(t *testing.T) {
 		if numOutboundSDSUpdate <= 1 {
 			t.Errorf("Number of SDS updates at outbound cluster should be greater than one, get %d", numOutboundSDSUpdate)
 		}
-		if numInboundSDSUpdate <= 1 {
-			t.Errorf("Number of SDS updates at inbound listener should be greater than one, get %d", numInboundSDSUpdate)
+		if numInboundSDSUpdate <= 0 {
+			t.Errorf("Number of SDS updates at inbound listener should be greater than zero, get %d", numInboundSDSUpdate)
```

Comment:
Why is this safe to change?

Reply:
numInboundSDSUpdate = len(certSet) - 1. So if we only receive one unique cert we will throw an error as it is not updated at all. 

As from the flaky failed test log: 
` TestCertRotation: rotation_test.go:96: Number of SDS updates at inbound listener should be greater than one, get 1`
the numInboundSDSUpdate = 1 which means we get at least one update of inbound. And other test stats are successful and no need to be skipped

So by easing a little of the criteria we can get such good advantages compared to just skip the test:
1. other test stats will be kept
2. the flaky failed case can be avoided
3. we can at least guarantee one rotation of inbound cert succeed

Reply:
numInboundSDSUpdate was defined as the total number of SDS pushes.
Perhaps we can rename them to numInboundSDSRotate, and numOutboundSDSRotate and use
if numInboundSDSRotate < 1 { t.Errorf() } if numOutboundSDSRotate < 1 { t.Errorf() }.